config-freebsd.md

```diff
@@ -0,0 +1,124 @@
+# <a name="FreeBSDContainerConfiguration" />FreeBSD Container Configuration
+
+This document describes the schema for the [FreeBSD-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+
+## <a name="configFreeBSDDevices" />Devices
+
+Devices in FreeBSD are accessed via the `devfs` filesystem. Typically, each container will have a `devfs` filesystem mounted into its `/dev` directory. Often, a minimal set of devices is exposed to the container using ruleset 4 from `/etc/defaults/devfs.rules` - the ruleset is specified as a mount option.
```

Comment:
Do we want to have a SHOULD here ("containers SHOULD have a `devfs` filesystem mounted into its `/dev` directory")?